### ISSUE TITLE
testStructTokens2() is defined twice in BitStream unittest.  First one fails but never executes.

### ISSUE THREAD

**Issue by FlyingLotus1983** [scott-griffiths/bitstring#166]
testStructTokens2() is defined twice in `test_bitstream.py`.  The second one on line 2486 over-writes the first declaration on line 2456.

If the first method is changed to `testStructTokens3()`, it will fail with an AssertionError:

```
======================================================================
FAIL: testStructTokens3 (__main__.Split)
----------------------------------------------------------------------
Traceback (most recent call last):
  File "C:\Users\njjansen\Downloads\bitstring-master\bitstring-master\test\test_bitstream.py", line 2470, in testStructTokens3
    self.assertEqual(pack('@h', 23), BitStream('intbe:16=23'))
AssertionError: BitStream('0x1700') != BitStream('0x0017')
```

Looking at the tests in the two methods, it does appear that they do different jobs and I think the first one failing is legitimate, indicating a bug in decoding `intbe` type BitStream objects when endianness is 'big'.

Commenting out the offending line 2470 and re-running, lines 2471-2475 also fail (must comment each out one at a time as the first failure will immediately bomb). So the assertions in questions are:

```
        self.assertEqual(pack('@h', 23), BitStream('intbe:16=23'))
        self.assertEqual(pack('@H', 23), BitStream('uintbe:16=23'))
        self.assertEqual(pack('@l', 23), BitStream('intbe:32=23'))
        self.assertEqual(pack('@L', 23), BitStream('uintbe:32=23'))
        self.assertEqual(pack('@q', 23), BitStream('intbe:64=23'))
        self.assertEqual(pack('@Q', 23), BitStream('uintbe:64=23'))
```

..all cause failures.

I'm running Python 3.5.1 on Windows x86-64.  I don't think this is a show-stopper for me, but figured I'd pass the information along.

Thanks!

**Comment by FlyingLotus1983** [comment]
Thanks @scott-griffiths for the quick response.  I just updated the main description for formatting and spelling errors

**Comment by scott-griffiths** [comment]
Thanks for that. Probably got hidden in a code reorganisation - I must think of better method names. I'll take a look properly when I switch back to this project.

**Comment by scott-griffiths** [comment]
Fixed on 3.1-update branch.
